Sources/Apodini/Request/Dynamics.swift

```diff
@@ -0,0 +1,26 @@
+//
+//  Dynamics.swift
+//  
+//
+//  Created by Max Obermeier on 10.12.20.
+//
+
+import Foundation
+
+public struct Dynamics {
```

Comment:
Please make it a sequence or collection... To make iterating over the injectables easier

Reply:
I think it makes sense to make the `Dynamics`' contents accessible as a `Collection`, however, a user is probably only interested in elements of a specific type. Thus I wouldn't make `Dynamics` itself a collection but provide a generic property/function that returns a `Collection` that only contains the elements which conform to that generic type.